easysocial/easysocial/group_members.php

```diff
@@ -107,16 +107,16 @@ public function getGroup_Members()
 	/**
 	 * Method this common function is for getting dates for month,year,today,tomorrow filters.
 	 *
-	 * @param   string  $group_id  group id
-	 * @param   string  $filter    filter name
-	 * @param   int     $log_user  logged user id
-	 * @param   string  $mapp      mapp object
+	 * @param   string  $clusterId  cluster id
```

Comment:
@Pratikshakhandagale  -  `$clusterId` param seems to be of type INT and not string